lib/RGBot.js

```diff
@@ -421,6 +421,37 @@ class RGBot extends EventEmitter {
         return null
     }
 
+    /**
+     * Gets the team name for this bot
+     * @return {string|null} Name of the team that this bot is on
+     */
+    myTeam() {
+        return this.teamForPlayer(this.username())
+    }
+
+    /**
+     * Returns a list of all players on the same team as this bot.
+     * @param {boolean} includeSelf if true, includes this bot in the list
+     * @return {string[]} A list of all player usernames on the same team as this bot.
+     */
+    getTeammateUsernames(includeSelf) {
+        const myTeam = this.myTeam();
+        return this.#matchInfo.players
+            .filter(player => player.team === myTeam && (includeSelf || player.username !== this.username()))
+            .map(player => player.username)
+    }
+
+    /**
+     * Returns a list of all players on the enemy team (relative to this bot).
+     * @return {string[]} A list of all player usernames on the enemy team.
+     */
+    getEnemyUsernames() {
```

Comment:
Personal preference...

```suggestion
    getOpponentUsernames() {
```

Enemy to me means any enemy (mob/player/etc), whereas opponent is the other players